src/Microsoft.Diagnostics.Runtime.Utilities/DbgEng/Wrappers/IDebugAdvancedWrapper.cs

```diff
@@ -17,6 +19,42 @@ int IDebugAdvanced.GetThreadContext(Span<byte> buffer)
                 return vtable->GetThreadContext(self, ptr, buffer.Length);
         }
 
+        int IDebugAdvanced.GetCommentWide(out string? comment)
+        {
+            GetVTable(this, out nint self, out IDebugAdvancedVtable* vtable);
+            comment = null;
+            char[] buffer = ArrayPool<char>.Shared.Rent(1048576/2);
+            try
+            {
+                fixed (char* ptr = buffer)
+                {
+                    DEBUG_READ_USER_MINIDUMP_STREAM minidumpStream = default;
+                    minidumpStream.StreamType = MINIDUMP_STREAM_TYPE.CommentStreamW;
+                    minidumpStream.Flags = 0u;
+                    minidumpStream.Offset = 0u;
+                    minidumpStream.Buffer = new IntPtr(ptr);
+                    minidumpStream.BufferSize = (uint)(buffer.Length * sizeof(char));
+                    minidumpStream.BufferUsed = 0u;
+                    HResult hr = vtable->Request(self, (uint)DEBUG_REQUEST.READ_USER_MINIDUMP_STREAM, &minidumpStream, sizeof(DEBUG_READ_USER_MINIDUMP_STREAM), null, 0, null);
+                    if (hr < 0)
```

Comment:
This should follow the standard .Net coding convention.  If any part of an if needs braces, all of them should have braces.  Additionally, `else` should not be on a different line than `if`.  e.g.:

``` C#
if (hr < 0)
{
    return hr;
}
else if (minidumpStream.BufferUsed > 1)
{
    comment = new string(ptr, 0, ((int)minidumpStream.BufferUsed - 2) / 2);
    return hr;
}
else
{
    comment = string.Empty;
}
```

Reply:
Done